lessons/lesson3.md

```diff
@@ -0,0 +1,39 @@
+## Lesson 3: Verify the result
+
+### Deep dive in
+Now let's take a deeper look into these actions:
+- `bound_package` : default param package created as binding of the shared package deployed in adobe namespace and having the validate action inside itself. Now, this bound_package will also have the validate_action residing in it due to binding.
+- `acp` - package created which will keep the new sync_event_handler sequence
+- `sync_Event_handler_7Z5KH5vv6X` - the new event handler unique to this registration with webhook url
+- `3rd_party_custom_events_3C9419175E9D393C0A495E39@AdobeOrg_2a0237a4-f0d3-45e9-8179-10ab21ef929c_eventrt_7Z5KH5vv6X` - the user sequence created taking the validate action as first action and user runtime action as second action. Created with the same unique identifier suffixed at the end of its name to bind this user sequence with the event registration in 1:1 fashion.
+-  `publish-events` - the user business logic runtime action
+
+
+### Verfiy the result
+Now, we will ingest some events to see how this new SYNC webhook behaves, send through `publish-event` or using postman 
+After ingesting the event, this is the request to the sync webhook, I made on successful one and another failed one to show the debug tracking feature
+![debug-13](assets/debug-tracing-13.png)
+
+ Now we can see json response returned from the main event handler webhook in the debug tracing
+![debug-14](assets/debug-tracing-14.png)
+
+
+The debug tracing response in case of user action code failure. Note that, now the json response includes the activation id of the failed user_sequence (using which user can trace it down to his action code failure) along with the exact user code failure message.
```

Comment:
I could see the event payload in the failure case. Why does it fail?

Reply:
How it fail does not matter as long as you made on failed call, this display that we have this debug tracing feature which could return meaningful activation id,  the return of action's main function is captured in Debug Tracing's Response Body now. previous the payload only return empty information,  see this https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/webhook/runtime_webhooks.md#built-in-signature-verification

Reply:
Just to close the loop - I made a new comment here: https://github.com/AdobeDocs/adobeio-codelabs-event-runtime-action-webhook/pull/1#discussion_r676711686.